core/modules/qdisp/QueryRequest.cc

```diff
@@ -33,6 +33,7 @@
 #include "qdisp/QueryRequest.h"
 
 // System headers
+#include <cstddef>
```

Comment:
I wondered if anyone would ask :). I added it when doing original migration and then John touch these lines so it got removed during rebasing. I figured since we are using nullptr in that file, so it'd be good to keep the include, although clearly some other header is bringing it. I am ok with removing that include if you think it is better.